projects/packages/videopress/src/client/admin/components/video-card/types.ts

```diff
@@ -24,6 +24,16 @@ export type VideoCardProps = Pick< VideoPressVideo, 'title' | 'plays' | 'id' > &
 		 */
 		loading?: boolean;
 
+		/**
+		 * True when is in uploading mode.
+		 */
+		uploading?: boolean;
+
+		/**
+		 * True when is in processing mode.
+		 */
+		processing?: boolean;
+
```

Comment:
There is a `isDeleting` and a `loading`. Maybe `isDeleting` should be renamed to `deleting` for consistency